<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PrintTo</class>
 <widget class="QDialog" name="PrintTo">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>730</width>
    <height>187</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>730</width>
    <height>187</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>730</width>
    <height>187</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Print To Document/Paper</string>
  </property>
  <property name="windowIcon">
   <iconset resource="runge.qrc">
    <normaloff>:/Runge/Resources/16x16/printer.png</normaloff>:/Runge/Resources/16x16/printer.png</iconset>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>627</x>
     <y>20</y>
     <width>91</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxOpenAfterSave">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>411</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Open exported document</string>
   </property>
   <property name="text">
    <string>Open document a&amp;fter save</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>161</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Print To</string>
   </property>
   <widget class="QRadioButton" name="radioButtonPrintToPDF">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Export current drawing to PDF file</string>
    </property>
    <property name="text">
     <string>P&amp;DF file</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPrintToPS">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>141</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Export current drawing to PS file (Not available yet in Qt5)</string>
    </property>
    <property name="text">
     <string>Post&amp;Script file</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonPrintToPaper">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>82</width>
      <height>24</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>Print current drawing to paper</string>
    </property>
    <property name="text">
     <string>&amp;Paper</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxDocumentSize">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>291</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>PDF/PS document size</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>20</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>Document Si&amp;ze</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>comboBoxDocumentSize</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditMarksLength">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>100</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Axes marks length (in coordinate units)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Axes &amp;marks length</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEditMarksLength</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditHandleDiameter">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>100</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Solution handles diameter (in coordinate units)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>100</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Handle diameter</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEditHandleDiameter</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Axes &amp;line width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEditAxesWidth</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditAxesWidth">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Axes lime width (in coordinate units)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditSolutionsWidth">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>60</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Solutions line width (in coordinate units)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>60</y>
     <width>141</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Sol&amp;utions line width</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buddy">
    <cstring>lineEditSolutionsWidth</cstring>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>radioButtonPrintToPDF</tabstop>
  <tabstop>radioButtonPrintToPS</tabstop>
  <tabstop>radioButtonPrintToPaper</tabstop>
  <tabstop>comboBoxDocumentSize</tabstop>
  <tabstop>lineEditAxesWidth</tabstop>
  <tabstop>lineEditMarksLength</tabstop>
  <tabstop>lineEditSolutionsWidth</tabstop>
  <tabstop>lineEditHandleDiameter</tabstop>
  <tabstop>checkBoxOpenAfterSave</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="runge.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PrintTo</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>619</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>330</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PrintTo</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>632</x>
     <y>118</y>
    </hint>
    <hint type="destinationlabel">
     <x>332</x>
     <y>172</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPrintToPaper</sender>
   <signal>toggled(bool)</signal>
   <receiver>checkBoxOpenAfterSave</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>65</x>
     <y>117</y>
    </hint>
    <hint type="destinationlabel">
     <x>83</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonPrintToPaper</sender>
   <signal>toggled(bool)</signal>
   <receiver>comboBoxDocumentSize</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>110</y>
    </hint>
    <hint type="destinationlabel">
     <x>344</x>
     <y>37</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
